sxs/utilities/lvcnr/waveforms.py

```diff
@@ -2,7 +2,8 @@
 
 import numpy as np
 import h5py
-
+import sxs
```

Comment:
Doesn't this cause a circular import issue?  I think you probably want to use `from ... import load` (or so) inside the `convert_modes` function.